config/src/config/secure_config.rs

```diff
@@ -18,6 +18,12 @@ pub struct SecureConfig {
     pub key_manager: KeyManagerConfig,
 }
 
+impl SecureConfig {
```

Comment:
Should we eliminate SecureConfig and just have KeyManagerConfig? What are we getting out of this layer of abstraction?

Reply:
My thinking was that we'll probably need some future config support for other secure components in the future, hence why I didn't just specify the key manager alone. If this assumption is wrong, we can clean it up later on (we still have to clean up some of the config anyway -- e.g., make the KeyManagerConfig a high-level component)